guides/common/modules/proc_creating-a-composite-content-view.adoc

```diff
@@ -5,17 +5,20 @@ Use this procedure to create a composite Content View.
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-composite-content-view_{context}[].
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create New View*.
-. In the *Name* field, enter a name for the view.
+. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create content view*.
+. In the *Create content view* pop-up window, enter a name for the view in the *Name* field.
```

Comment:
I'm not sure if we should use "pop-up window", maybe I'd go with just "window".
(I haven't found any other usage of "pop-up window" in the docs apart from this PR.)